frame/dapps-staking/src/pallet/mod.rs

```diff
@@ -819,13 +915,13 @@ pub mod pallet {
             RegisteredDapps::<T>::insert(contract_id.clone(), developer.clone());
             RegisteredDevelopers::<T>::insert(&developer, contract_id.clone());
 
-            // create new ContractEraStake item
-            let era_staking_points = EraStakingPoints {
-                total: <BalanceOf<T>>::default(),
-                stakers: BTreeMap::new(),
-            };
-            let current = Self::current_era().unwrap_or(Zero::zero());
-            ContractEraStake::<T>::insert(&contract_id, &current, era_staking_points);
+            // create new ContractEraStake item // TODO: Why is this needed here? Contract wasn't staked, it was only registered.
```

Comment:
@Maar-io please take a look at this part.

Reply:
@Dinonard true, not needed here, since this is handled in the bond_and_stake()